routes/index.js

```diff
@@ -61,7 +98,7 @@ router.post('/', upload.single('file'), function (req, res, next) {
         };
         params.Key = directoryPath;
         params.Body = stringifiedJson;
-        client.upload(params, function uploadCallback(err, data) { // upload to S3
+        client.upload(params, function uploadCallback(err, data) {
```

Comment:
Handle error. Hope this is done after functionality completion. Merging for now.